core/server/master/src/main/java/alluxio/master/file/meta/MountTable.java

```diff
@@ -436,19 +491,53 @@ public UfsManager.UfsClient getUfsClient(long mountId) {
   /**
    * Resolves the given Alluxio path. If the given Alluxio path is nested under a mount point, the
    * resolution maps the Alluxio path to the corresponding UFS path. Otherwise, the resolution is a
-   * no-op.
+   * no-op. It will call {@link MountTable#resolve(AlluxioURI, List)}.
```

Comment:
```
   * Resolves the given Alluxio path. If the given Alluxio path is nested under a mount point, the
   * resolution maps the Alluxio path to the corresponding UFS path. 
   * If the full path exists, the search will utilize the `MountTable` and therefore be fast.
   * Otherwise, the search is based on the URI string and be slower.
```